### PR TITLE
Fix and add a way to test labeling

### PR DESCRIPTION
**Pull request by dynbit**

### CODE REVIEW

File: VelocitySDK/Core/VLTApiClient.m [hunk 1]
```diff
@@ -19,7 +19,8 @@
 #import "VLTCore.h"
 #import "VLTMotionDetectResult.h"
 
-static NSString * const VLTApiClientBaseUrl = @"https://sdk.vlcty.net/api/";
+//static NSString * const VLTApiClientBaseUrl = @"https://sdk.vlcty.net/api/";
+static NSString * const VLTApiClientBaseUrl = @"http://192.168.2.35:8080/api/";
```
Comment (Antanas) [inline VelocitySDK/Core/VLTApiClient.m]: leftover

File: VelocitySDK/Core/VLTApiClient.m [hunk 2]
```diff
@@ -160,7 +161,7 @@ - (void)uploadMotionData:(nonnull VLTPBCapture *)capture
                                                  parameters:params
                                                        data:data
                                                       error:&error];
-    if (req != nil) {
+    if (req == nil) {
```
Comment (Antanas) [inline VelocitySDK/Core/VLTApiClient.m:163]: Does this means that upload didn't work before?

File: VelocitySDK/VLTManager.m [hunk 3]
```diff
@@ -158,10 +158,18 @@ + (void)markGoalAsCompleted:(nonnull NSString *)goalId
...
+                    completion:(nonnull void(^)(BOOL success))completion {
     [[VLTManager shared].client pushMotionDataWithLabels:labels
-                                                 success:^{}
+                                                 success:^{
+                                                     vlt_invoke_block(completion, YES);
+                                                 }
                                                  failure:^(NSError * _Nullable error) {
-                                                     NSLog(@"[VelocitySDK] error: markCurrentMotionAs %@ failed.", labels);
+                                                     vlt_invoke_block(completion, NO);
+                                                     NSLog(@"[VelocitySDK] error: %@ markCurrentMotionAs %@ failed. ", error, labels);
```
Comment (Antanas) [inline VelocitySDK/VLTManager.m:172]: We'll have to think about logging in the future as SDK users might not want to see our logs

**Review by Antanas — COMMENTED** [review]
No need to bump sdk version? As there was a fix

File: VelocitySDK/VLTManager.m (hunk 3 above)
Comment (dynbit) [reply VelocitySDK/VLTManager.m:172]: Yes.

File: VelocitySDK/Core/VLTApiClient.m (hunk 2 above)
Comment (dynbit) [reply VelocitySDK/Core/VLTApiClient.m:163]: Yes.

**Pull request merged**